test/parallel/test-debugger-extract-function-name.js

```diff
@@ -20,18 +20,22 @@ cli.waitForInitialBreak()
   .then(() => cli.command('exec a = function func() {}; a;'))
   .then(() => assert.match(cli.output, /\[Function: func\]/))
   .then(() => cli.command('exec a = function func () {}; a;'))
-  .then(() => assert.match(cli.output, /\[Function\]/))
+  .then(() => assert.match(cli.output, /\[Function: func\]/))
   .then(() => cli.command('exec a = function() {}; a;'))
-  .then(() => assert.match(cli.output, /\[Function: function\]/))
+  .then(() => assert.match(cli.output, /\[Function: \(anonymous\)\]/))
```

Comment:
There is a problem that when the function has no explicit name, `node inspect` can't get the function name from RemoteObject.
That is, the name of `a = function() {}` should be `a` (node display `[Function: a]` as a result of `a = function() {}; a`) but now it become `anonymous` with `node inspect`.